run_scripts/04_fetch_surveys.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 if [ $# -ne 5 ]; then
-    echo "Usage: sh 01_fetch_messages.sh <user> <rapid-pro-root> <rapid-pro-server> <rapid-pro-token> <data-root>"
```

Comment:
When we get back to the same place, I'd like to chat about how we can parameterise these a bit - or at least understand some of where these come from and how much will be different with the templates.